data/quarg/quarg outfits.txt

```diff
@@ -113,11 +115,12 @@ outfit "Medium Graviton Thruster"
 	"mass" 70
 	"outfit space" -70
 	"engine capacity" -70
-	"thrust" 80
+	"thrust" 85
 	"thrusting energy" 20
-	"thrusting heat" 10
+	"thrusting heat" 3
+	"force protection" 0.1
```

Comment:
Force protection of 0.1 is quite low. Does this actually have a noticeable effect on ships as small as Quarg ones?
Also consider adding drag reduction which was merged recently? #7485